news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -26,96 +27,98 @@ First of all, congratulations to the winners of the April 2017 Spotlights. They
 
 ## osu!mania
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/trigger.jpg)](https://osu.ppy.sh/s/555076)
 
-I have to say it. This map is definetly the most fun experience I had in a while from the ranked pool and I can only say thanks and congratulations to Wonki for winning this month spotlights and also for providing us such a good piece. The SVs on this map feels intense but they are not overdone and it just feels natural with the song, making it really confortable to play. Combine that with a cool usage of minijacks at the very end with some simple LN patterns this map just has everything you want to be the map of the month... and its art!
+This beatmap set by **[Wonki](https://osu.ppy.sh/users/7898495)** definitely is a fun experience from the ranked pool and congratulations to Wonki for winning this month spotlights and also for providing us such a good piece. The scroll speed changes on this map feels intense but they are not overdone and it just feels natural with the song, making it really confortable to play. Combine that with a cool usage of minijacks at the very end with some simple long note patterns this map just has everything you want to be the map of the month... and its art!
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/re-lm.jpg)](https://osu.ppy.sh/s/588066)
 
-This map caught my eye for a few reasons. The most obvious being the heavy use of SV, which is thoughtfully crafted to be intuitive and smooth, which is quite impressive given how easy it is to cross the line of being more of a distraction than a feature. Though the SV is not the main reason I think this map is particularly deserving of a spotlight. Lude doesn't shy away from LN usage in their mapping of this track. Instead, they use it to add atmospheric value, complexity and originality, which is what I personally love to see. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves, makes this a suitable pick for May's spotlight.
+This map is eye-catching for a few reasons: The most obvious being the heavy use of scroll speed variations, which are thoughtfully crafted to be intuitive and smooth. This is quite impressive given how easy it is to cross the line of being more of a distraction than a feature. However, the scroll speed variations are not the main reason this map is particularly deserving of a spotlight. **[Lude](https://osu.ppy.sh/users/6974536)** doesn't shy away from long note usage in their mapping of this track. Instead, they use it to add atmospheric value, complexity, and originality, which is great to see in the ranked section. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves makes this a suitable pick for May's spotlight.
```

Comment:
no capitalization after colons unless they're semicolons, which it probably should be here.